### PR TITLE
Lallstedters patch 1

### PR DESCRIPTION
**Pull request by lallstedters**
## Describe your changes

Updated info for Adas page, including relevant names and contact info.

## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

File: en/clubs/geek/body.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ The purpose of this club is to make it easier for people identifying as women or
 
 <br>
 
+**Ada 22/23*
```
Comment (Herkarl) [inline en/clubs/geek/body.md:7]:
```suggestion
**Ada 22/23**
```

File: en/clubs/geek/body.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ The purpose of this club is to make it easier for people identifying as women or
...
+Klara Sandström - *Adas Ordförande (aka head of Ada)* <br />
+Ebba bråtman <br />
+Isadora Winter <br />
+Alva Välimäki <br />
+Sandra Reinecke <br />
+
 **Ada 21/22*
```
Comment (Herkarl) [inline en/clubs/geek/body.md:14]:
```suggestion
**Ada 21/22**
```

**Review by Herkarl — CHANGES_REQUESTED** [review]
Liten formateringssak, men annars LGTM

File: en/clubs/geek/sidebar.md [hunk 3]
```diff
@@ -2,12 +2,9 @@
 
 Send us an email or follow us on facebook if you find us interesting or would like to know more!
 
-Öfvermatrona, 
-
-Amanda Hallstedt
+Head of Ada,
```
Comment (elmaxe) [inline en/clubs/geek/sidebar.md:5]:
```suggestion<br />
Head of Ada,
```

File: en/clubs/geek/sidebar.md [hunk 4]
```diff
@@ -2,12 +2,9 @@
...
+Klara Sandström
```
Comment (elmaxe) [inline en/clubs/geek/sidebar.md:6]:
```suggestion
Head of Ada,<br />
Klara Sandström
```

**Review by elmaxe — CHANGES_REQUESTED** [review]

**Comment by niklasvatn** [comment]
@Herkarl @elmaxe @lallstedters 
I committed the suggested edits.
Merging it would be good. Might want to double-check with the author first though.

**Comment by lallstedters** [comment]
> 

I'm fine with the changes, thank you for fixing it and sorry for the sloppy work.

**Review by elmaxe — APPROVED** [review]

**Review by Herkarl — APPROVED** [review]

**Pull request merged**